config/defaults.conf

```diff
@@ -54,191 +60,225 @@ local_log_path = ${run_dir}"/log.log"
+                                     // file when starting do_target_task_training.
+allow_untrained_encoder_parameters = 0  // Set for experiments involving random untrained encoders 
+                                        // only. Allows do_target_task_training and do_full_eval 
+                                        // to proceed without pretraining.
+allow_reuse_of_pretraining_parameters = 0  // Set to 1 to allow task models that were trained 
+                                           // during pretraining from being reused in 
+                                           // do_target_task_training. This may behave incorrectly 
+                                           // if a run is stopped and restarted in 
+                                           // do_target_task_training (issues #285, #290).
 allow_missing_task_map = 0  // Dangerous: If true, ignore missing classifier_task_map.json
-                            // This is needed for bare-ELMo probing, since the pretraining phase is skipped for these models.
-reload_tasks = 0     // If true, force the rebuilding of the task files in the experiment directory, even if they exist.
-reload_indexing = 0  // If true, force the rebuilding of the index files in preproc/ for tasks in reindex_tasks, even if they
-                     // exist.
-target_train_data_fraction = 1  // Use only the specified fraction of the training data in the do_target_task_training phase.
-                        // should not impact pretraining, even for the same task.
-                        // pretrain_data_fraction should not be used together with                 // target_train_data_fraction
-                        // both should not be less then 1 at a same time
-reindex_tasks = ""
-
-reload_vocab = 0     // If true, force the rebuilding of the vocabulary files in the experiment directory. For classification and
+                            // Only relevant to runs that use ELMo. This is needed for bare-ELMo 
+                            // probing, since the pretraining phase is skipped for these models.
+reload_tasks = 0  // If true, force the rebuilding of the task files in the experiment directory, 
+                  // even if they exist.
+reload_indexing = 0  // If true, force the rebuilding of the index files in preproc/ for tasks in 
+                     // reindex_tasks, even if they exist.
+reindex_tasks = ""  // See reload_indexing above.
+reload_vocab = 0     // If true, force the rebuilding of the vocabulary files in the experiment 
+                     // directory. For classification and
                      // regression tasks with the default ELMo-style character handling, there is no vocabulary.
-
-is_probing_task = 0  // Set if the tasks in target_tasks are NLI-style probing tasks. (Ellie: Is this right?)
+is_probing_task = 0  // Set if the tasks in target_tasks are NLI-style probing tasks.
```

Comment:
weird for this to be a config option and not a task attr

Reply:
Also might be worth double-checking if it's actually used anywhere/fully supported, else say that it's only partially supported.

Reply:
This is a real (highish-priority, I think) issue, but not one I can resolve here: #372.